### ISSUE TITLE
remove unused ORC code

### ISSUE THREAD

**Issue by thinrope** [georgmartius/vid.stab#95]
I have been looking around and there are is quite a pile of ORC code that per your comment is not used:

https://github.com/georgmartius/vid.stab/blob/master/CMakeLists.txt#L32
````
### ORC is not used in any active code at the moment  ###
# I tried it with 0.4.14
#  0.4.10 did not work (not all opcode implemented)
# find_package(Orc)
````
I am talking about the snippet above, as well as `src/orc` and `test/orc_bug` and some references in the makefiles.

**Comment by georgmartius** [comment]
True, it was not useful and will not become useful again. If you feel like creating a merge request -- always welcome ;-)

**Comment by georgmartius** [comment]
Done — the ORC code was removed in 05d2903. `src/orc`, `tests/orc_bug` and the CMake references are all gone from master.
